packages/sites/genomics-site/webapp/wdkCustomization/js/client/routes.jsx

```diff
@@ -161,6 +162,26 @@ function DownloadsRouteComponent() {
  * Wrap Ebrc Routes
  */
 export const wrapRoutes = (ebrcRoutes) => [
+  {
+    path: '/search/dataset/AllDatasets/result',
+    requiresLogin: false,
+    component: () => (
+      <AnswerController
+        {...{ question: 'AllDatasets', recordClass: 'dataset' }}
+      />
```

Comment:
This would do the same thing:

```suggestion
      <AnswerController
        question="AllDatasets"
        recordClass="dataset"
      />
```

Reply:
It would be good to add a comment above these new entries something like this:

```
// Allow guests to access All Datasets for SEO and public visibility
```